var/spack/repos/builtin/packages/unigen/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Unigen(MakefilePackage):
+    """The United Generators project was launched by the Virtual Institute 146
+    VI-SIM in September 2005 following a proposal of Herbert Strobele.
+    The goal was to facilitate comparison between various models (see below)
+    and/or various experiments (HADES, FOPI, CERES, NA49, CBM). The package
+    at present allows to convert output of various event generators to a
+    generic root format."""
+
+    homepage = "https://www.gsi.de/work/wissenschaftliche_netzwerke/helmholtz_virtuelle_institute/unigen.htm"
+    url      = "https://github.com/FairRootGroup/UniGen/archive/v2.3.tar.gz"
+

```

Comment:
```suggestion


    maintainers = ['iarspider']
```

Reply:
I mean, I don't want to add myself as a maintainer - not now.

Reply:
Thanks for the clarification!